src/main/java/org/schemaspy/output/dot/schemaspy/DotTableFormatter.java

```diff
@@ -206,39 +206,43 @@ private static Factory getFactory(Table table, boolean includeExcluded) {
         return factory;
     }
 
-    private static Set<Table> getTableImmediateRelatives(Table table, Factory factory, boolean includeImplied) {
-        Columns columns = factory.columns();
-        Set<TableColumn> relatedColumns = new HashSet<>();
+    private static Set<Table> immediateRelativesTables(Table table, Factory factory, boolean includeImplied) {
+        Set<Table> relatedTables = new HashSet<>();
+        relatedTables.addAll(childrenTables(factory, includeImplied));
+        relatedTables.addAll(parentsTables(factory, includeImplied));
+        relatedTables.remove(table);
+        return relatedTables;
+    }
 
+    private static Set<Table> childrenTables(Factory factory, boolean includeImplied) {
+        Columns columns = factory.columns();
+        Set<Table> result = new HashSet<>();
```

Comment:
Result as a name doesn't convey what the result is or what it suppose to represent.
childTables?

Reply:
I argue that `result` is clear. It communicates what will become the output of the method, so it isn't confused with temporary variables. Please note that the method already contains e.g., `children` and `childColumn`. Then again, all of this is consequences of too little object collaboration.